src/js/modules/infragistics.ui.combo.js

```diff
@@ -32,6 +32,7 @@
 			"jquery-ui",
 			"./infragistics.util",
```

Comment:
If widget depends on util do we need to specify the combo depends on both util and widget?